issue-147: show the correct response with nanosecond precision (#155)
VictoriaMetrics/victorialogs-datasource: CHANGELOG.md

* FEATURE: add compatibility for Grafana `v10.x.x` to ensure `/select/logs/hits` displays precise logs volume on the Explore page. See [this comment](https://github.com/VictoriaMetrics/victorialogs-datasource/pull/146#issuecomment-2533419498).

> * BUGFIX: fix rounding of the timestamp if response from datasource contains nanoseconds. See [this issue](https://github.com/VictoriaMetrics/victorialogs-datasource/issues/147).

Author: yeah, it should be milliseconds precision :). In the first iteration, I wrote about milliseconds after changes to nanos

> * BUGFIX: fix rounding of the timestamp if response from datasource contains milliseconds. See [this issue](https://github.com/VictoriaMetrics/victorialogs-datasource/issues/147).

Reviewer: ```suggestion
* BUGFIX: properly parse timestamps with milliseconds precision in datasource response. See [this issue](https://github.com/VictoriaMetrics/victorialogs-datasource/issues/147).
```
